go/chat/storage/storage.go

```diff
@@ -442,6 +445,11 @@ func (s *Storage) updateAllSupersededBy(ctx context.Context, convID chat1.Conver
 		}
 	}
 
+	// conv source will queue asset deletions in the background
+	if convSource := s.G().ConvSource; convSource != nil {
```

Comment:
Why do this?

Reply:
`ConvSource` isn't set on `g` in the storage tests. Should I add it to the test setup?